package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "opendistro-alerting",
-  "version": "1.0.0.0",
+  "version": "1.1.0.0",
```

Comment:
Can you remove these changes from this PR.

Reply:
sure